nautobot/cloud/models.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+
+from nautobot.core.constants import CHARFIELD_MAX_LENGTH
+from nautobot.core.models.generics import OrganizationalModel
+from nautobot.extras.utils import extras_features
+
+
+@extras_features(
+    "custom_links",
+    "custom_validators",
+    "dynamic_groups",
+    "export_templates",
+    "graphql",
+    "webhooks",
+)
+class CloudAccount(OrganizationalModel):
+    name = models.CharField(max_length=CHARFIELD_MAX_LENGTH, help_text="The name of this Cloud Account.")
+    description = models.CharField(max_length=CHARFIELD_MAX_LENGTH, blank=True)
+    account_number = models.CharField(max_length=CHARFIELD_MAX_LENGTH, help_text="The number of this Cloud Account.")
+    provider = models.ForeignKey(to="dcim.Manufacturer", on_delete=models.PROTECT, related_name="cloud_accounts")
```

Comment:
Just out of interest, this means that it would be normal to create a `dcim.Manufacturer` called "AWS" and "Azure" to associate with cloud components?

Reply:
Yes. This is also in support of being able to use the Device model to track resources like EC2 instances.